src/assets/js/google.js

```diff
@@ -0,0 +1,18 @@
+document.addEventListener('DOMContentLoaded', () => {
+    if (window.location.hostname === 'localhost') {
+        return;
+    }
+    const id = 'YOUR_ID_HERE';
```

Comment:
I assume this will be changed at build time.

Reply:
Currently file is not used